### PR TITLE
Add documentation for NNI PAI mode experiment

### PR DESCRIPTION
**Pull request by yds05**
1. Add documentation for NNI PAI mode experiment
2. Update v0.1 to v0.2 in all documents

### CODE REVIEW

File: docs/PAIMode.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+**Run an Experiment on Multiple Machines**
```
Comment (QuanluZhang) [inline docs/PAIMode.md]: 'Multiple Machines' -> 'PAI'?

File: docs/PAIMode.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# choice: local, remote, pai
+trainingServicePlatform: pai
+# choice: true, false  
+useAnnotation: true
+tuner:
+  builtinTunerName: TPE
+  classArgs:
+    optimize_mode: maximize
+trial:
+  command: python mnist.py
```
Comment (chicm-ms) [inline docs/PAIMode.md]: python3 ?

Comment (yds05) [reply docs/PAIMode.md]: Resolved

File: docs/PAIMode.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
+**Run an Experiment on OpenPAI**
+===
+NNI supports running an experiment on [OpenPAI](https://github.com/Microsoft/pai) (aka pai), called pai mode. Before starting to use NNI pai mode, you should have an account to access an [OpenPAI](https://github.com/Microsoft/pai) cluster. See [here](https://github.com/Microsoft/pai#how-to-deploy) if you don't have any OpenPAI account and want to deploy an OpenPAI cluster. In pai mode, your trial program will run in pai's container created by Docker.
```
Comment (scarlett2018) [inline docs/PAIMode.md]: How about change the 1st sentence to:
Starting from v0.2.0, NNI supports one more Training Service Mode: pai mode, which enabled user to run an experiment on [OpenPAI](https://github.com/Microsoft/pai).

File: docs/PAIMode.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+* dataDir
+    * Optional key. It specifies the HDFS data direcotry for trial to download data. The format should be something like hdfs://{your HDFS host}:9000/{your data directory}
+* outputDir 
+    * Optional key. It specifies the HDFS output direcotry for trial. Once the trial is completed (either succeed or fail), trial's stdout, stderr will be copied to this directory by NNI sdk automatically. The format should be something like hdfs://{your HDFS host}:9000/{your output directory}
+
+Once complete to fill nni experiment config file and save (for example, save as exp_pai.yaml), then run the following command
+```
+nnictl create --config exp_pai.yaml
+```
+to start the experiment in pai mode. NNI will create OpanPAI job for each trial, and the job name format is something like `nni_exp_{experiment_id}_trial_{trial_id}`.
```
Comment (scarlett2018) [inline docs/PAIMode.md]: OpanPAI is a typo.

File: docs/PAIMode.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+  gpuNum: 0
+  cpuNum: 1
+  memoryMB: 8196
+  image: openpai/pai.example.tensorflow
+  dataDir: hdfs://10.1.1.1:9000/nni
+  outputDir: hdfs://10.1.1.1:9000/nni
+# Configuration to access OpenPAI Cluster
+paiConfig:
+  userName: your_pai_nni_user
+  passWord: your_pai_password
```
Comment (scarlett2018) [inline docs/PAIMode.md]: we have to expose the password here?

File: docs/PAIMode.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+
+Once a trial job is completed, you can goto NNI WebUI's overview page (like http://localhost:8080/oview) to check trial's information. 
+
+Expand a trial information in trial list view, click the logPath link like:
+![](./nni_webui_joblist.jpg)
+
+And you will be redirected to HDFS web portal to browse the output files of that trial in HDFS:
+![](./nni_trial_hdfs_output.jpg)
+
+You can see there're three fils in output folder: stderr, stdout, and trial.log
```
Comment (scarlett2018) [inline docs/PAIMode.md]: "fils" is a typo.

**Pull request merged**